app/models/user.rb

```diff
@@ -33,4 +33,10 @@ def is_member_or_admin?
     admin? || is_member?
   end
 
+  def is_company_owner?(this_company_number)
```

Comment:
But I do need a similar thing, because I don't want the link to be shown to *any* user who is associated with a company. I want the link to be shown to the users associated with *this particular* company